phpunit.mongo.xml

```diff
@@ -13,7 +13,7 @@
         <env name="SYMFONY_DEPRECATIONS_HELPER" value="weak_vendors" />
         <server name="KERNEL_DIR" value="tests/Fixtures/app/" />
         <server name="KERNEL_CLASS" value="AppKernel" />
-        <server name="APP_ENV" value="test_mongodb" />
+        <server name="APP_ENV" value="mongodb" />
```

Comment:
@alanpoulain is this OK ? I had to update the env so the filters are declared in phpunit too, I didn't understood exactly why it was setup like that ?

Reply:
to me yes it should be `mongodb`

Reply:
It shouldn't be `mongo` IMO. It's `test_mongo` (`config_test_mongo.yml`) to match the `test` (`config_test.yml`) in `phpunit.xml`. I don't understand why you need to change it.

Reply:
```
APP_ENV=mongodb ./vendor/bin/behat --stop-on-failure --suite=mongodb features/main/input_output.feature
```

Is how I launch mongodb tests?

Reply:
mhh check the travis config APP_ENV is set to `mongodb`, the config is set to `config_mongodb.yml`, are you sure?

Reply:
We need it to be `test_mongodb` to be equivalent to the ORM part. Or we change also the ORM part (to `postgres` for instance).

Reply:
it has been reverted @alanpoulain as discussed